### ISSUE TITLE
训练过程中准确率一直没有变化

### ISSUE THREAD

**Issue by cc-cv** [shartoo/luna16_multi_size_3dcnn#12]
在每个ct图像上我取了100个反例，跑了1000个epoch，准确率一直是0.593750，loss也一直基本没有什么变化。请问是什么问题呢？您试验的效果怎么样

**Comment by shartoo** [comment]
嗯，我也遇到这个问题，目前在排查中。。

**Comment by shartoo** [comment]
我怀疑这个论文是不是有问题，我尝试了很多修改loss都没法下降。

**Comment by jieshaoxxiansen** [comment]
> 我怀疑这个论文是不是有问题，我尝试了很多修改loss都没法下降。

我有一个疑问，在你提取数据的时候，你将annotations.csv提取为正标签，将candidates_V2.csv提取为负标签，您是否注意到candidates_V2.csv文件中已经包含了annotations.csv的全部标签

**Comment by liuchuangjun321** [comment]
loss不收敛代码在交叉熵使用的时候有个错误：
cross_entropy=tf.reduce_sum(tf.nn.softmax_cross_entropy_with_logits(logits=net_out,label = real_label))
改为cross_entropy=tf.nn.softmax_cross_entropy_with_logits(logits=net_out,label = real_label)

**Comment by shartoo** [comment]
@jieshaoxxiansen 你说的我倒真没注意到，我以为candidates_v2.csv全部是负样本标签呢。感谢提醒。
@liuchuangjun321 你说的这个我之前也改过好像也没法纠正问题。
